README.md

```diff
@@ -1,15 +1,21 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# React Template
+# Stock Market Watcher
 
-> This is a template design to start react projects fast. It already contains all the most commonly used development packages. scss and js linters installed as well. 
+> Stock Market Watcher is an App that allows you to consult the market cap and the stock price of different companies on the NASDAQ and NYSE. 
 
 ## Built With
 
 - ReactJS
 - HTML
 - JavaScript
 
+## Check the live demo
+- [Stock Market Watcher Demo](https://john-arboleda.github.io/stock-market-data/)
+
+## Presentation
+- [Click here](https://www.loom.com/share/667831a827a142f099e657bebc1cb590) to watch a description of the project.
+
```

Comment:
- [ ] Kindly make sure your README contains an `Acknowledgements` section that gives appropriate credit to Nelson Sakwa the author of the original design, as required. 👍🏽

Reply:
Thank you for taking the time to review a project submitted this late. 🙏🏿